addons/test/misra/misra-test.c

```diff
@@ -287,7 +287,7 @@ void misra_7_4(void)
    misra_7_4_call(1, "text_call"); // 7.4 11.8
 }
 
-bool test(char *a)
+bool test_8_2(char *a)
```

Comment:
CI still fails because this line generates a violation for 2.7 rule. You could fix it by adding ` // 2.7`.